src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/ClientConfigurationSettings.java

```diff
@@ -167,6 +167,16 @@ public static class ClientConfigurationFormatSettings {
         public @NotNull OnTypeFormattingSettings onTypeFormatting = new OnTypeFormattingSettings();
     }
 
+    /**
+     * Client-side references settings.
+     */
+    public static class ClientConfigurationReferencesSettings {
```

Comment:
And can be configured via client config for user-defined language server definitions. It's _disabled by default_ here as well.